app/views/LocationShare/LiveLocationPreviewModal.tsx

```
@@ -0,0 +1,527 @@
+import React, { useState, useEffect, useRef } from 'react';
+import { View, Text, TouchableOpacity, StyleSheet, ActivityIndicator, Alert, Linking, InteractionManager } from 'react-native';
+import { Image as ExpoImage, type ImageErrorEventData } from 'expo-image';
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)